lib/src/text-input/TextInput.tsx

```diff
@@ -500,6 +502,7 @@ const DxcTextInput = React.forwardRef<RefType, TextInputPropsType>(
                   }}
                   backgroundType={backgroundType}
                   tabIndex={tabIndex}
+                  title="Increment"
```

Comment:
Same as above. Maybe something like "Increment number" would be more useful to the user.

Reply:
Changed by "Increment value".